src/pages/loading-waiting-states/progress-bar/index.tsx

```diff
@@ -0,0 +1,234 @@
+        setUploadFileList((oldList) => [...oldList, createFileItem()]);
+    }, [list, setUploadFileList]);
+
+    useEffect(() => {
+        const progressInterval = setInterval(() => {
+            const newList = [...list];
+            for (let i = 0; i < list.length; i++) {
+                if (list[i].progress < 100) {
+                    const newItem: FolderItem = {
+                        ...list[i],
+                        progress: list[i].progress + 1,
+                        status: `Uploading (${list[i].progress + 1})`,
+                    };
+                    newList[i] = newItem;
+                } else {
+                    const newItem: FolderItem = {
+                        ...list[i],
+                        status: `Complete`,
+                    };
+                    newList[i] = newItem;
+                    setTimeout(() => {
+                        setUploadFileList((oldList) => oldList.filter((item) => item.id !== list[i].id));
+                    }, 10000);
+                }
+            }
+            setUploadFileList(newList);
+        }, 100);
+        if (list.length < 1) {
+            clearInterval(progressInterval);
+        }
+        return (): void => {
+            clearInterval(progressInterval);
+        };
+    }, [list]);
+
+    return (
+        <div style={{ minHeight: '100vh' }}>
+            <AppBar data-cy="pxb-toolbar" position={'sticky'}>
+                <Toolbar classes={{ gutters: classes.toolbarGutters }}>
+                    <Hidden mdUp={true}>
+                        <IconButton
+                            data-cy="toolbar-menu"
+                            color={'inherit'}
+                            onClick={(): void => {
+                                dispatch({ type: TOGGLE_DRAWER, payload: true });
+                            }}
+                            edge={'start'}
+                            style={{ marginRight: 20 }}
+                        >
+                            <MenuIcon />
+                        </IconButton>
+                    </Hidden>
+                    <Typography variant={'h6'} color={'inherit'}>
+                        Progress Bars
+                    </Typography>
+                    <div />
+                </Toolbar>
+            </AppBar>
+            <div className={classes.container}>
+                <div className={classes.uploadButtonContainer}>
+                    <Button
+                        variant={'contained'}
+                        color={'primary'}
+                        startIcon={<VerticalAlignTop />}
+                        onClick={uploadFile}
+                    >
+                        <Typography noWrap color={'inherit'}>
+                            UPLOAD NEW FILE
+                        </Typography>
+                    </Button>
+                </div>
+                <Card>
+                    <FormControl className={classes.formControlStyle} component="fieldset">
```

Comment:
For this simple card it doesn't matter much, but for keeping in line with recommended best practices, let's wrap the content here in a CardContent